php/laravel/README.md

```diff
@@ -0,0 +1,54 @@
+# Build and run
+
+## Configuring the project to use YugabyteDB Cluster
+
+update the `.env` file for configuring the database to connect with YugabyteDB cluster started in the previous step.
+
+```
+DB_CONNECTION=pgsql
+DB_HOST=127.0.0.1
+DB_PORT=5433
+DB_DATABASE=yugabyte
+DB_USERNAME=yugabyte
+DB_PASSWORD=
+```
+
+## Setup the database to use Laravel migrations
+
+```
+cd yb-laravel-example/
```

Comment:
I gather that users can either run the below commands in current directory or inside yb-laravel-example/. But one cannot know how to create yb-laravel-example/ without looking for it in Laravel.md